### ISSUE TITLE
posXY unit is 'tiles' even if room is in meters

### ISSUE THREAD

**Issue by hchybz** [ditrit/OGrEE-3D#216]
values are ok.. not unit

``` 
+ro:UDS.QC1.MG.A104A@[12.4,6.6]@[12.6,5.4,0.4]@+N+W@m
//
// Racks storage (m)
//
+rk:UDS.QC1.MG.A104A.QTE01@[1.2,2]@lenovo-everyscale-48u@front
+rk:UDS.QC1.MG.A104A.QTE02@[1.8,2]@lenovo-everyscale-48u@front
+rk:UDS.QC1.MG.A104A.QTE03@[2.4,2]@lenovo-everyscale-48u@front
```

![image](https://user-images.githubusercontent.com/61938544/170678953-9995a053-6f6d-45e1-89e3-4d67013e6fe3.png)

**Comment by Cedrok** [comment]
It's a CLI issue: by the *posXYUnit* is "tile" by default.  
We need to retrieve the *floorUnit* attribute from parent room and map it to *posXYUnit*.
```
room.attributes[floorUnit] = "t" => rack.attributes[posXYUnit] = "tile"
room.attributes[floorUnit] = "m" => rack.attributes[posXYUnit] = "meter"
room.attributes[floorUnit] = "f" => rack.attributes[posXYUnit] = "foot"
```
@denizul : It's on your side

**Comment by denizul** [comment]
We have 2 fixes, one on the API (issue216AlwaysTiles 613d9751ccc) and one on the CLI (issue100_posXY 6aca2f9fc). We believe the fix on the CLI to be the better one to use
